packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -356,6 +360,7 @@ class IconButton extends StatelessWidget {
         autofocus: autofocus,
         canRequestFocus: onPressed != null,
         onTap: onPressed,
+        onLongPress: onLongPress,
```

Comment:
line 357 and line 314 may also need to be updated.
bonus question: is it ok that a iconbutton has onLongPressed but not onPressed?

Reply:
Thanks for the review.

I think it's ok to have onLongPressed but not onPressed.